### DISCUSSION TITLE
Change or Modify Attributes Name

### DISCUSSION THREAD

**Discussion by dkusumah** [spatie/laravel-activitylog#847] [Q&A] [answered]
Hi,

Sorry if my question has been asked before, I've been digging to the docs and discussions but can't find the topic like my issue here.

Is there a simple way to change the attributes name I have into a specific name while using the `tapActivity()`? I.e. I have attributes like 

```
protected static $logAttributes = [
        'claim_no', 'date_advise', 'claim_detail',
        'insured_pic'
];
```
And will be logged into properties field like :

`{ Claim No: 12345, Date Advise: 20 Aug 2021, Claim Detail: Car Accident, Insured PIC: Matt Hammer }`

Many thanks to anyone who can help me.

DK

**Comment by Gummibeer** [comment, marked as answer]
This was asked a few times ago - at least once. But can't remember the issue number.
My recommendation is to do this during data display on runtime instead of during save. But for sure you can walk through the array and change all keys in the `tapActivity()` method.

**Reply by dkusumah** [reply]
Ah yes, I missed that one. I thought there was a simple approach for my issue, but nevermind I will try your advice. Thanks for helping me out. :beers:
